src/database/DataTypes/User.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
```

Comment:
If I was reviewing, I might annoyingly point out that the template should be updated to `2020` :)